package.json

```diff
@@ -10,6 +10,10 @@
     "removeOracleLimit": "node ./removeMaxAllowedLimit.js",
     "moveFR": "node ./moveFR.js",
     "setFee": "node ./setFees.js",
+    "setTrading": "node ./setTrading.js",
+    "setMarginBankWithdrawal": "node ./setMarginBankWithdrawal.js",
+    "setFinalSettlementStatus": "node ./setFinalSettlementStatus.js",
```

Comment:
rename the script to `enableFS` and file to `enableFinalSettlement.ts` you can only enable it, and nothing else.